src/less/resets.less

```diff
@@ -1,9 +1,9 @@
-@import (reference) "./animations";
-
+@import (reference) "../less/index";
```

Comment:
i think this could end up in circular references - can remove, since it's unused